### ISSUE TITLE
[FR] Add transparency to hs.chooser

### ISSUE THREAD

**Issue by NightMachinary** [Hammerspoon/hammerspoon#2583]
I like to add transparency to the background of hs.chooser so that I can see what's underneath it.

**Comment by AlexZhong22c** [comment]
something like this? https://github.com/Hammerspoon/Spoons/issues/196

**Comment by NightMachinary** [comment]
> the background should already be translucent like other macOS windows (like the Finder sidebar)... are you talking about full transparency?

Yes, real transparency without the blur effect so that I can see (read) what’s behind the dialogue.
